Lib/test/test_doctest.py

```diff
@@ -3218,19 +3218,8 @@ def load_tests(loader, tests, pattern):
     return tests
 
 
-def test_coverage(coverdir):
-    trace = import_helper.import_module('trace')
-    tracer = trace.Trace(ignoredirs=[sys.base_prefix, sys.base_exec_prefix,],
-                         trace=0, count=1)
-    tracer.run('test_main()')
-    r = tracer.results()
-    print('Writing coverage results...')
-    r.write_results(show_missing=True, summary=True,
-                    coverdir=coverdir)
-
-
 if __name__ == '__main__':
-    if '-c' in sys.argv:
-        test_coverage('/tmp/doctest.cover')
-    else:
-        unittest.main()
```

Comment:
With
```diff
index e51f54ef35..e530767e66 100644
--- a/Lib/test/test_doctest.py
+++ b/Lib/test/test_doctest.py
@@ -3219,7 +3219,6 @@ def load_tests(loader, tests, pattern):
 
 
 if __name__ == '__main__':
-    raise RuntimeError(
-        "Running `test_doctest` via cmdline is not supported, "
-        "use `-m test test_doctest` instead"
-    )
+    __name__ = 'test.test_doctest'
+    __import__(__name__)
+    unittest.main()
```
I have only one test failure.  Why not keep #111348 for this issue (removing ``test_coverage()`` does make sense for me) and use ``unittest.main()``?

Reply:
Nice alternative to changing the tests!